backend/src/recommitment/recommitment.service.ts

```diff
@@ -251,7 +251,10 @@ export class RecommitmentService {
         };
 
       default:
-        return null;
+        return {
```

Comment:
What does this change affect? I don't really see a consequence in this pr

Reply:
If the default case is chosen an error was previously thrown since we're destructing the returned value.

Reply:
Mostly this was to clean up the code in the useSUpe hook